src/i18n/locales/en/seatLayout.json

```diff
@@ -0,0 +1,5 @@
+{
+    "available": "Booked",
+    "unavailable": "Unavailable",
+    "selected": "Selected"
```

Comment:
better to have single json file en.json and put  page items as objects

Reply:
Sera has made this change in her PR: https://github.com/amaljoeqb/bus-ticket-booking-react/pull/13